persistence-utils-product/product.json

```diff
@@ -12,10 +12,21 @@
           },
           {
             "groupId": "com.axonivy.utils.persistence",
-            "artifactId": "persistence-utils-test",
+            "artifactId": "persistence-utils-demo-tool",
             "version": "${version}",
-            "type": "iar",
-            "importInWorkspace": "false"
+            "type": "iar"
+          },
+          {
+            "groupId": "com.axonivy.utils.persistence",
+            "artifactId": "persistence-utils-demo-test",
```

Comment:
do you really want that this will be imported when installing persistence utils?
we only do that in our demo-projects to demonstrate how you could write tests with axon ivy.

Reply:
prior "importInWorkspace"was set to false. so it won't be imported by default, you need to click on a checkbox, if you really want it.

Reply:
I've seen that you skip deployment of artifactory, so this wont work.. I would remove it from here.